kasa/smartcamera.py

```diff
@@ -0,0 +1,130 @@
+"""Module for SmartCameras (EC70)."""
+from typing import Any, Dict, Optional
+
+from .credentials import Credentials
+from .modules import PTZ, Cloud, Time
+from .modules.ptz import Direction, Position
+from .smartcameraprotocol import SmartCameraProtocol
+from .smartdevice import DeviceType, SmartDevice, requires_update
+
+
+class SmartCamera(SmartDevice):
+    """Representation of a TP-Link Kasa Camera.
+
+    To initialize, you have to await :func:`update()` at least once.
+    This will allow accessing the properties using the exposed properties.
+
+    All changes to the device are done using awaitable methods,
+    which will not change the cached values, but you must await :func:`update()`
+    separately.
+
+    Errors reported by the device are raised as
+    :class:`SmartDeviceExceptions <kasa.exceptions.SmartDeviceException>`, and should be
+    handled by the user of the library.
+
+    Examples:
+        >>> import asyncio
+        >>> camera = KasaCam("127.0.0.1")
+        >>> asyncio.run(camera.update())
+        >>> print(camera.alias)
+        Camera2
+
+        Cameras, like any other supported devices, can be turned on and off:
+
+        >>> asyncio.run(camera.turn_off())
+        >>> asyncio.run(camera.turn_on())
+        >>> asyncio.run(camera.update())
+        >>> print(camera.is_on)
+        True
+    """
+
+    def __init__(
+        self,
+        host: str,
+        credentials: Credentials,
+        *,
```

Comment:
```suggestion
        *,
        credentials: Credentials,
```
For consistency, all other classes expect it as kwarg-only.